client/src/components/DisplayTable/DisplayTable.stories.tsx

```diff
@@ -27,16 +31,23 @@ const common_column_configs = {
   price: {
     index: 1,
     header: "Price",
-    is_summable: true,
+    is_sortable: true,
   },
-};
+} as unknown as ColumnConfigProps;
```

Comment:
Should drop this. Same as before, this is actually forcing it to cast to this type even though it doesn't actually satisfy it. The proper type checking should happen inside `Basic.args` below.

Reply:
Going to have the type system complaining about a bunch of missing fileds in `common_column_configs` once you do that and the proper type checking applies. Either the typing on `DisplayTable` is bad (missing optional flags again) or this config is actually incomplete. 

Edit: oh, yup, the `DisplayTable` typing is junk, my bad for letting that get past review. A bunch of column config props that have defaults are still marked as required. Part of a history of us doing a bad job of typing React components with default props, refer to this if you're going to try to fix it https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/default_props/